CHANGELOG.md

```diff
@@ -64,6 +37,34 @@ preflightTest.on('completed', (report: PreflightTestReport) => {
   console.log(" Your estimated quality score was " + report.qualityScore);
 });
 ```
+
+The [PreflightTestReport](https://sdk.twilio.com/js/video/releases/2.17.0/docs/global.html#PreflightTestReport) generated by `preflightTest` on `completed` provides statistics that can be used to determine the quality of the connection. Some of the useful stats and their optimal values are listed below.
+
+| Stat                     |  Description | Optimal values
+| -------------------------|--------------|------------------|
+| jitter                   | Packets delay variation on audio tracks | < 30ms
+| packetLoss               | Packet loss as a percent of total packets sent | < 1%
+| rtt                      | Round trip time in milliseconds. | < 200ms
+
+The API also calculates an overall `qualityScore` using jitter, packet loss, and rtt values observed. The quality score can be one of:
+
+| Value                    | Meaning
+| -------------------------|--------------------------------|
+| 0                        | Failed
+| 1                        | Degraded
+| 2                        | Fair
+| 3                        | Good
+| 4                        | Great
+| 5                        | Excellent
+
+
+A few things to note:
+
+- This function uses <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API"> web audio API's.</a> Browser's autoplay policies sometimes require user action before accessing these APIs. Please ensure that this API is called in response to user action like a button click.
+- `preflightTest` emits a `failed` event to indicate test failures. You can use the Preflight `progress` events to better understand where the test failed and refer to
+[this guide](https://www.twilio.com/docs/video/build-js-video-application-recommendations-and-best-practices#connection-errors) for interpreting common errors.
+- Stats are based on the P2P connection. This means that the `outgoingBitrate` and `incomingBitrate` returned in the `stats` object are not a measure of uplink / downlink bandwidth. 
```

Comment:
This bullet is a little confusing. For this loopback test, should the outgoingBitrate equal the incomingBitrate?

Reply:
yes, outgoing and incoming will  be mostly equal (excluding some packet loss).  @donaltoomey  Are you suggesting we just report overall bitrate instead of these two separate values?

Reply:
yeah, I think it is tied to my other question above about the 2 peer connections. At the end of the day we need to provide some indication of what the upstream bitrate is going to be like and what the downstream bitrate is going to be like when the participant connects to an actual Room. For example, if they are able to get say 1 Mbps upstream and downstream then they should be able to have a good Video call.

Reply:
Preflight test  publishes one audio and video track. This does not give good indication of bandwidth availability. I think customer needs separate bandwidth test. I believe we already have such test in Diagnostic SDK that uses data channels - Or am I missing something @charliesantos ? )

Reply:
ok, we may need a call at some stage to discuss. I think what we really want to tell end users is what kind of bitrate they can expect when connected to the Twilio Cloud. Speed tests likely have edge caches and use TCP so are not really representative of what video/audio bitrate you are going to get.

Reply:
Right, we have bitrate tests that uses data channels (between to peer connections) while connected to Twilio Turn Servers.